packages/flutter/lib/src/rendering/custom_paint.dart

```diff
@@ -611,10 +614,12 @@ class RenderCustomPaint extends RenderProxyBox {
   ) {
     assert(() {
       if (child == null && children.isNotEmpty) {
-        throw FlutterError(
-          '$runtimeType does not have a child widget but received a non-empty list of child SemanticsNode:\n'
-          '${children.join('\n')}'
-        );
+        throw FlutterError.fromParts(<DiagnosticsNode>[
```

Comment:
I think we missed this one, this one is changing everything to be summary, is this intended?
@jacob314

Reply:
good catch.  Only the first line should be the summary.